tests/unit/amo/components/TestSearch.js

```diff
@@ -32,6 +32,7 @@ describe(__filename, () => {
 
   beforeEach(() => {
     props = {
+      context: VIEW_CONTEXT_EXPLORE,
```

Comment:
You probably know this but we should always dispatch actions for property values that are set by `mapStateToProps`. The older pattern of setting mapped prop values manually moves us away from reality since it only simulates how the application changes state.

It's not worth a follow-up patch; I know this was an older test that did not use `shallowUntilTarget()` and fixing it would be more effort. For future situations like this, I'd suggest trying to fix old test suites as part of the patch as long as it's not too time consuming (some are harder to convert than others).

Other than this, the patch looks good. Sorry I didn't get a chance to review it in time.

Reply:
Thanks 👍